ES/ESmy018.xml

```diff
@@ -250,7 +345,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      erasures. The name of Zäbänä Krǝstos (<locus target="#8ra">8ra</locus>) is
```

Comment:
this statement should be moved to the history part of the description
correct the spelling